lib/Settings/ProfilesLabel/tests/ProfileLabel-test.js

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import {
+  describe,
+  beforeEach,
+  afterEach,
+  it,
+} from '@bigtest/mocha';
+import { expect } from 'chai';
+
+import { mountWithContext } from '../../../../test/bigtest/helpers';
+import {
+  ProfilesLabelInteractor,
+  ProfilesPopoverInteractor,
+} from './interactors';
+import ProfilesLabel from '../ProfilesLabel';
+
+import translations from '../../../../translations/stripes-data-transfer-components/en';
+
+describe('ProfilesLabel', () => {
```

Comment:
Should we have `rendering` part for consistency? I see that we have the wording like that for the majority of components.
```suggestion
describe('rendering ProfilesLabel', () => {
```

Reply:
It is consistent with other modules. Please, take a look a the line 23 and other components

Reply:
You are right.